config/conversations/questions/whats_for_lunch.rb

```diff
@@ -0,0 +1,99 @@
+require "net/https"
+
+Houston::Slack.config do
+  listen_for(/what.*s for lunch(?:\s(?:on\s)?(?<modifier>next\s)?(?<day>.*))?\??/i) do |e|
+    day = e.match[:day] || "today"
+    day = day.downcase.gsub(/\?/, "")
+    day = "today" unless day =~ /^(?:mon|tues|wed(?:nes)?|thu(?:rs)?|fri)(?:day)?$|^tomorrow$/
+
+    target_date = Date.today
+    target_date += 1.day if day == "tomorrow"
+    unless day =~ /today|tomorrow/
+      find_next_week = !e.match[:modifier].nil?
+      target_wday = case day
+      when /mon/
+        1
+      when /tue/
+        2
+      when /wed/
+        3
+      when /thu/
+        4
+      when /fri/
+        5
+      end
+      today_wday = target_date.wday
+      if today_wday > target_wday
+        day_diff = 7 - (today_wday - target_wday)
+        target_date += day_diff.days
+      elsif today_wday == target_wday && find_next_week
+        target_date += 7.days
+      else
+        day_diff = target_wday - today_wday
+        target_date += day_diff.days
+      end
+    end
+
+    e.reply "Hold on, I'll check..."
+
+    Houston.side_projects.start! Houston::SideProject::Lunch.new(
+      user: e.user,
+      conversation: e.start_conversation!,
+      date: target_date)
+  end
+end
+
+module Houston
+  module SideProject
+    class Lunch < Base
+      attr_reader :date
+
+      FULLDATE = "%A, %B %-d, %Y"
+
+      def initialize(attributes)
+        @date = attributes.fetch :date
+        super attributes.merge(description: "I'm looking up what's for lunch on #{date.strftime(FULLDATE)}.")
+      end
+
+      def start!
+        fetch_lunch_menu
+      end
+
+      def fetch_lunch_menu
+        http = Net::HTTP.new "cphweb09", 80
+        request = Net::HTTP::Get.new '/mycph/menu.asp'
+        response = http.request request
```

Comment:
You should use [Faraday](https://github.com/lostisland/faraday) instead of `Net::HTTP`. `net/http` in Ruby's standard library has a pretty terrible interface (the above took 3 lines, there are inconsistencies between constructing a GET and a POST, etc) and I almost never use it anymore. Houston uses Faraday extensively and you can count on it as a dependency.